.travis.yml

```diff
@@ -40,6 +40,7 @@ services:
 addons:
   postgresql: "9.6"
 script:
+  - mix compile --warnings-as-errors
```

Comment:
Will this also fail with deprecation warnings? For example if `mix deps.get` pulls in an updated dependency and the compiler raises a warning because of a deprecated method?

I know that it doesn't break if the deprecation warning comes _from a dependency_, but I'm thinking about deprecation warnings coming from FunWithFlags itself.